### DISCUSSION TITLE
Get-FalconHost - select only specific fields

### DISCUSSION THREAD

**Discussion by andreiv3103** [CrowdStrike/psfalcon#217] [Q&A] [answered]
Hello
Is there a way to make Get-FalconHost return only specific fields I need for each host. I don't need all the data the -Detailed parameter returns.
I know how to select post command, but it would be very useful not to have to select post Get-FalconHost and insted get directly what I need.
Thanks.

**Comment by Sl1m007** [comment, marked as answer]
Hi @andreiv3103,

If you are creating your own script, you can use the following '| select-object hostname, etc, etc'.
This will only work with '-Detailed' used.

**Reply by andreiv3103** [reply]
Hello.
Yes, I know how to use the Select-Object cmdlet. I just thought there is way to request specific fields directly from the Get-FalconHost command.
Thank you!

**Reply by bk-cs** [reply]
@Sl1m007 - Great answer, thanks!

@andreiv3103 - If you're asking "can I request that the API only send specific fields"; no--that is not possible. `Select-Object` is the only way to do it (i.e. dropping the fields you don't care about after receiving them from the API).
